src/cascadia/TerminalApp/Utils.h

```diff
@@ -30,3 +30,26 @@ inline std::string JsonKey(const std::string_view key)
 }
 
 winrt::Windows::UI::Xaml::Controls::IconElement GetColoredIcon(const winrt::hstring& path);
+
+// This is a pair of helpers for determining if a pair of guids are equal, and
+// establishing an ordering on GUIDs (via std::less).
+namespace std

```

Comment:
An alternative is to declare a Comparison struct and use it at declaration.   Adding this to the std namespace feels yucky.    Seems to be semi-acceptable though.